packages/core/card.ts

```diff
@@ -249,6 +251,10 @@ export class Card {
     return cardDocumentFromJsonAPI(jsonapi);
   }
 
+  get cardDir(): string | undefined {
```

Comment:
Is this the directory there the module lives for the card? or is this something specific to just cards that come from the git realm. If this is purely a git realm need, then let's not do this here.

Reply:
> Is this the directory where the module lives for the card

Yes, this was at @ef4 's suggestion, so that cards are loaded from the local fs if they are present there, and the data is stored on the card - this allows using of npm modules in cards which was not working previously.